packages/slice-machine/server/src/api/index.ts

```diff
@@ -64,11 +64,7 @@ router.get(
   ): Promise<Express.Response> {
     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
     const payload = await state(req);
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-    if (payload.clientError) {
```

Comment:
how are errors sent to the client now?

Reply:
The error are not sent anymore but in the client side we never use the fact that this call can send a different status code that's why I removed it.